packages/shared/src/components/auth/OnboardingLogs.tsx

```diff
@@ -112,6 +112,7 @@ const TiktokTracking = (): ReactElement => {
 
 interface LogSignUpProps {
   experienceLevel: keyof typeof UserExperienceLevel;
+  isExtension?: boolean;
```

Comment:
Not sure if it's intended, but this has nothing to do with squad header right?

Reply:
Probably pulled from main, then pointed to this intermediate branch it is not yet updated.